sdk/mixedreality/Azure.MixedReality.RemoteRendering/src/Customizations/Models/RemoteRenderingModelFactory.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Azure.MixedReality.RemoteRendering
+{
+    /// <summary>
+    /// Model factory that enables mocking for the Remote Rendering library.
+    /// </summary>
+    public static class RemoteRenderingModelFactory
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AssetConversionOutput"/> class for mocking purposes.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
I don't think you need these EditorBrowsables.

Reply:
I'll remove them - I just replicated what I saw was done in other SDKs 😅